File: java-main/java-main/NestedforloopAssi/p23.java
/*
10
10 9
9 8 7
7 6 5 4
*/

class Demo{

        public static void main(String[]args){

                int N=4;
                int num=10;

                for(int i=1;i<=N;i++){
                        for(int j=1;j<=i;j++){
                                System.out.print(num-- + " ");
                        }
                        num++;
                        System.out.println();
                }
        }

}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C+DS/codes/Recursion/p23.c
/* Count occurence of zero in given number */
# include <stdio.h>
int czero (int num) {
	int count=0 ;
	while (num!=0) {
		int rem=num%10 ;
		if (rem==0) 
			count++ ;
		num=num/10 ;
	}
	return count ;
}
void main () {
	int num ;
	printf("Enter number\n") ;
	scanf("%d",&num) ;

	if (num==0) 
		printf("zero occures 1 time\n") ;
	else {
		int ret=czero(num) ;
		printf("zero occures %d time\n",ret) ;
	}
}


